### PR TITLE
docs-cn for instance scoped system variables

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

It is translated from [docs/pull#7897](https://github.com/pingcap/docs/pull/7897)

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7897
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -5,13 +5,11 @@ aliases: ['/docs-cn/dev/system-variables/','/docs-cn/dev/reference/configuration
 
 # 系统变量
 
-TiDB 系统变量的行为与 MySQL 相似但有一些不同，变量的作用范围可以是全局范围有效 (Global Scope)、实例级别有效 (Instance Scope) 或会话级别有效 (Session Scope)，或组合了上述多个范围。其中：
+TiDB 系统变量的行为与 MySQL 相似，变量的作用范围可以是全局范围有效 (Global Scope) 或会话级别有效 (Session Scope)。其中：
```
Comment (TomShawn) [inline system-variables.md:8]:
```suggestion
TiDB 系统变量的行为与 MySQL 相似，变量的作用范围可以是会话级别有效 (Session Scope) 或全局范围有效 (Global Scope)。其中：
```

File: system-variables.md [hunk 2]
```diff
@@ -668,28 +730,40 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 > - 对于 v5.4.0 及以上版本的新建集群，该变量开关默认开启。
 
 - 作用域：SESSION | GLOBAL
-- 默认值：`ON`
+- 集群持久化：是
+- 默认值：`ON` 
 - 这个变量用于控制是否开启 index merge 功能。
 
+### tidb_enable_legacy_instance_scope <span class="version-mark">从 v6.0 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:737]:
```suggestion
### tidb_enable_legacy_instance_scope <span class="version-mark">从 v6.0.0 版本开始引入</span>
```

File: system-variables.md [hunk 3]
```diff
@@ -668,28 +730,40 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+
+- 作用域：SESSION | GLOBAL
+- 集群持久化：是
+- 默认值：`ON` 
+- 这个变量用于允许使用 `SET SESSION` 对 `INSTANCE` 作用域的变量进行设置，用法同 `SET GLOBAL`。
+- 为了兼容之前的 TiDB 版本，这个选项默认为 `ON`。
```
Comment (TomShawn) [inline system-variables.md:743]:
```suggestion
- 为了兼容之前的 TiDB 版本，该变量值默认为 `ON`。
```

File: system-variables.md [hunk 4]
```diff
@@ -1415,13 +1545,15 @@ SET tidb_query_log_max_len = 20;
 ### `tidb_retry_limit`
 
 - 作用域：SESSION | GLOBAL
+- 集群持久化：是
```
Comment (TomShawn) [inline system-variables.md:1548]: L1535 `tidb_replica_read` 漏加 “集群持久化：是”

File: system-variables.md [hunk 5]
```diff
@@ -1675,13 +1829,21 @@ set tidb_slow_log_threshold = 200;
...
 - 默认值：`SYSTEM`
 - 数据库所使用的时区。这个变量值可以写成时区偏移的形式，如 '-8:00'，也可以写成一个命名时区，如 'America/Los_Angeles'。
 - 默认值 `SYSTEM` 表示时区应当与系统主机的时区相同。系统的时区可通过 [`system_time_zone`](#system_time_zone) 获取。
 
+### `timestamp`
+
+- 作用域：SESSION
+- 默认值：`0`
+- 一个 Unix 时间戳，用于表示 `CURRENT_TIMESTAMP()`、`NOW()` 等函数的时间戳的一个非空值。该变量通常用于数据恢复或数据复制。
```
Comment (TomShawn) [inline system-variables.md:1841]:
```suggestion
- 一个 Unix 时间戳。变量值非空时，表示 `CURRENT_TIMESTAMP()`、`NOW()` 等函数的时间戳。该变量通常用于数据恢复或数据复制。
```

**Review by TomShawn — COMMENTED** [review]
Great job!

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**